composer.json

```diff
@@ -3,16 +3,17 @@
   "description": "PHP SDK for developing Nosto modules for e-commerce platforms",
   "license": "BSD-3-Clause",
   "require-dev": {
+    "codeception/base": "3.1.1",
     "codeception/specify": "^0.4.6",
     "phpmd/phpmd": "^2.6",
-    "squizlabs/php_codesniffer": "^2.6",
     "sebastian/phpcpd": "^3.0",
     "phing/phing": "2.*",
     "codeception/c3": "^2.0",
     "symfony/console": "3.*",
-    "wimg/php-compatibility": "^8.0",
-    "phan/phan": "^1.2",
-    "codeception/base": "3.1.1"
+    "phan/phan": "2.*",
+    "squizlabs/php_codesniffer": "^3.5",
+    "phpcompatibility/php-compatibility": "^9.0",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.5"
```

Comment:
Version `0.6[.2]` has come out in the mean time and it is highly recommended to use that instead. As Composer treats minors < 1.0 as majors, this has to be made explicit.

```
"dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.6"
```